core/eolearn/core/utils/testing.py

```diff
@@ -51,6 +53,7 @@ def generate_eopatch(
     """A class for generating EOPatches with dummy data."""
     config = config if config is not None else PatchGeneratorConfig()
     supported_feature_types = [ftype for ftype in FeatureType if ftype.is_array()]
+    supported_feature_types.append(FeatureType.META_INFO)
     parsed_features = FeatureParser(features or [], supported_feature_types).get_features()
```

Comment:
try using a callable for that. Probably `lambda fty: fty.is_array() or fty == FeatureType.META_INFO`?

Reply:
done